Create GSA-pages.md

JJediny opened the pull request:
## Changes proposed in this pull request:

- DRAFT for review and comment
- @drewbo 
- @kudehaatila 

## security considerations

Documents the steps required to request a new Authority to Use (ATU) for a GSA Website using https://pages.cloud.gov

JJediny committed:
Create tts-pages.md

drewbo reviewed (changes requested):
A few high level thoughts before I do a closer read:
- There is an existing page which covers [cloud.gov Pages itself](https://handbook.tts.gsa.gov/tools/pages/); I would definitely link those pages, or combine as appropriate.
- Prior to linking to the FedRAMP package, I would link to the page above as well as cloud.gov/pages so readers will have more understanding of the application and its purpose.

Let me know if you want help on either of those portions and I can add some language

JJediny committed:
Updates based on review and spellcheck

JJediny committed:
update-to-confirm-or-add-to-touchpoints

sknep reviewed (changes requested):
Some content changes would really help make this more readable and fit the cloud.gov content styleguide. Let me know if I can help! I don't mind making some of these changes for you if you're swamped.

JJediny committed:
Updates based on Feedback so far

TY @sknep @svenaas @drewbo

JJediny committed:
Update tts-pages.md

Spell check and internal link fix https://github.com/18F/handbook/actions/runs/8120129807/job/22196930149?pr=3814

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
rename back to gsa-pages

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
change request link

JJediny committed:
Merge branch 'gsa-pages-atu-WIP' of https://github.com/18F/handbook into gsa-pages-atu-WIP

JJediny committed:
minor edits

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
change blockqoutes to use handbook alert components where appropriate

JJediny committed:
Change capatalization based on feedback

https://github.com/18F/handbook/pull/3814#discussion_r1509252110

JJediny committed:
Change passive voice based on

https://github.com/18F/handbook/pull/3814#discussion_r1509262934

JJediny committed:
add to whom

https://github.com/18F/handbook/pull/3814#discussion_r1509261234

JJediny committed:
add to by whom

https://github.com/18F/handbook/pull/3814#discussion_r1509291010

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
swap banner order and remove multiple links to pages

JJediny committed:
remove more capitalization

drewbo committed:
use h4 for low systems alerts to avoid adding to in page nav

drewbo committed:
fix link, small formatting

KKAtila reviewed (commented):
The titles currently display "Create tts-pages.md", "TTS Pages",  "TTS Pages - Handbook Page". Please note the name change from TTS Pages to GSA Pages if it hasn't been updated already.

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
Delete tts-pages.md

JJediny committed:
Updates based on outreach feedback

JJediny committed:
Change the workflow

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
add mailto link

JJediny committed:
Merge branch 'gsa-pages-atu-WIP' of https://github.com/18F/handbook into gsa-pages-atu-WIP

JJediny committed:
add dns slack link

JJediny committed:
change to form entry

JJediny committed:
Merge branch 'main' into gsa-pages-atu-WIP

JJediny committed:
Add DLP link upfront

JJediny committed:
Merge branch 'gsa-pages-atu-WIP' of https://github.com/18F/handbook into gsa-pages-atu-WIP
